rain_api_core/logging.py

```diff
@@ -0,0 +1,261 @@
+import json
+import logging
```

Comment:
Can we rename logging.py to something that doesn't potentially conflict with the builtin?

Reply:
What would you suggest? I purposefully used the same name so that it would be clear that these are extensions to the standard logging module, and it's already in the `rain_api_core` namespace so there shouldn't be any conflicts.